egs/swbd/s5c/local/nnet3/run_lstm.sh

```diff
@@ -19,36 +31,43 @@ hidden_dim=1024
 recurrent_projection_dim=256
 non_recurrent_projection_dim=256
 chunk_width=20
-chunk_left_context=20
-clipping_threshold=5.0
+chunk_left_context=40
```

Comment:
This was a temporary thing until the state preservation feature was
implemented.

But I will try out chunk-width of 30 and left context of 20. With momentum
the instabilities were not seen. So I will just try this setting without
modifying chunks-per-minibatch or learning rate.



On Tue, Oct 6, 2015 at 4:49 PM, Daniel Povey <notifications@github.com>
wrote:

> In egs/swbd/s5c/local/nnet3/run_lstm.sh
> <https://github.com/kaldi-asr/kaldi/pull/214#discussion_r41320094>:
>
> > @@ -19,36 +31,43 @@ hidden_dim=1024
> >  recurrent_projection_dim=256
> >  non_recurrent_projection_dim=256
> >  chunk_width=20
> > -chunk_left_context=20
> > -clipping_threshold=5.0
> > +chunk_left_context=40
>
> OK. It might be worthwhile looking again at increasing the frames-per-eg,
> then (e.g. to 30?), and maybe reducing the minibatch size or learning rate
> slightly if you worry about stability (at least, until @xiaohui-zhang
> <https://github.com/xiaohui-zhang> checks in his changes to the
> max-change code). It just seems to me that having the left-context much
> more than the frames-per-eg is a bit wasteful.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/kaldi-asr/kaldi/pull/214/files#r41320094>.
>